db/model/Gdoc/archieToEnriched.ts

```diff
@@ -177,5 +178,16 @@ export const archieToEnriched = (text: string): OwidGdocContent => {
             ? htmlToSimpleTextBlock(citation)
             : citation.map(htmlToSimpleTextBlock)
     parsed.toc = toc
+    if (parsed.details) {
+        parsed.details = keyBy(
+            parsed.details.map(
+                (detail: any): { id: string; text: EnrichedBlockText[] } => ({
+                    id: detail.id,
+                    text: detail.text.map(parseRawBlocksToEnrichedBlocks),
```

Comment:
This means that we can, in theory, use full-fledged Archie syntax inside DoDs? Cool!

Reply:
you can even have a recursive dod 🙈